prow/oss/cluster/deck_private_deployment.yaml

```diff
@@ -0,0 +1,208 @@
+apiVersion: apps/v1
```

Comment:
nit: This file contains all the deck private resources, not just the deployment. Doesn't really matter though since we aren't using bazel to deploy.